templates/download/raspberry-pi/index.html

```diff
@@ -140,102 +140,84 @@ <h4 class="u-no-margin--bottom">Ubuntu {{ releases.lts.full_version }} LTS</h4>
-    <div class="col-3">
-      <h4>Ubuntu {{ releases.previous_lts.full_version }}</h4>
-      <p>The previous LTS version of Ubuntu for projects without 20.04 support.</p>
-    </div>
-    <div class="col-3">
-      <p class="u-hide--small u-sv2">&nbsp;</p>
-      <p>
-        <a href="/download/raspberry-pi/thank-you?version={{ releases.previous_lts.full_version }}&amp;architecture=armhf+raspi2" class="p-button--neutral" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Raspberry Pi 2 32-bit', 'eventLabel' : '{{ releases.previous_lts.short_version }}', 'eventValue' : undefined });">
-          Download 32-bit<span class="u-hide--medium u-hide--large"> for Raspberry Pi 2</span>
-        </a>
-      </p>
-    </div>
-    <div class="col-3">
-      <p>
-        <a href="/download/raspberry-pi/thank-you?version={{ releases.previous_lts.full_version }}&amp;architecture=arm64+raspi3" class="p-button--neutral" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Raspberry Pi 3 64-bit', 'eventLabel' : '{{ releases.previous_lts.short_version }}', 'eventValue' : undefined });">
-          Download 64-bit<span class="u-hide--medium u-hide--large"> for Raspberry Pi 3</span>
-        </a>
-      </p>
-      <p>
-        <a href="/download/raspberry-pi/thank-you?version={{ releases.previous_lts.full_version }}&amp;architecture=armhf+raspi3" class="p-button--neutral" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Raspberry Pi 3 32-bit', 'eventLabel' : '{{ releases.previous_lts.short_version }}', 'eventValue' : undefined });">
-          Download 32-bit<span class="u-hide--medium u-hide--large"> for Raspberry Pi 3</span>
-        </a>
-      </p>
-    </div>
-    <div class="col-3">
-      <p>
-        <a href="/download/raspberry-pi/thank-you?version={{ releases.previous_lts.full_version }}&amp;architecture=arm64+raspi4" class="p-button--neutral" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Raspberry Pi 4 64-bit', 'eventLabel' : '{{ releases.previous_lts.short_version }}', 'eventValue' : undefined });">
-          Download 64-bit<span class="u-hide--medium u-hide--large"> for Raspberry Pi 4</span>
-        </a>
-      </p>
-      <p>
-        <a href="/download/raspberry-pi/thank-you?version={{ releases.previous_lts.full_version }}&amp;architecture=armhf+raspi4" class="p-button--neutral" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Raspberry Pi 4 32-bit', 'eventLabel' : '{{ releases.previous_lts.short_version }}', 'eventValue' : undefined });">
-          Download 32-bit<span class="u-hide--medium u-hide--large"> for Raspberry Pi 4</span>
-        </a>
-      </p>
+    <div class="p-strip is-shallow">
+      <div class="u-fixed-width">
+        <hr>
+      </div>
     </div>
+    <div class="row u-equal-height">
```

Comment:
Should this also be conditional on the latest not being an LTS?

Reply:
again, the whole block is in a conditional